src/loader/implementation.ml

```diff
@@ -373,7 +188,7 @@ let postprocess_poses source_id poses uid_to_id uid_to_loc =
       (fun uid id acc ->
         let loc_opt = Shape.Uid.Tbl.find_opt uid_to_loc uid in
         match loc_opt with
-        | Some loc ->
+        | Some loc when not (LocHashtbl.mem loc_to_id loc) ->
```

Comment:
Is this to prevent generating overlapping anchors for the same loc ? A comment would be nice.